README.md

```diff
@@ -29,3 +30,14 @@ What is the asymptotic runtime ($\Theta$) of your algorithm in terms of the
 number of comparisons? What is it in terms of the number of flips? Add your
 answer to this markdown file.
 
+### Analysis
+
+The asymtotic runtime of my algorithm in terms of comparisons is $\Theta(n)$, because in the flip() function I make at most 2 comparisons to ensure I reverse the correct number of elements. In pancake sort I make 2 comparisons when finding the max unsorted element. One to verify if the current element is already sorted and the other to see if that element is larger than the max element. I also have the main while loop that will checks it's condition after every loop, this adds one more comparison. So total I have 5 comparisons per loop, and the algorithm loops until every element is sorted (sorting one element per loop) so 5n. Since we are looking at the runtime analysis we can drop the leading coefficient. Giving us with $\Theta(n)$ <br/> 
```

Comment:
It looks like there are two nested loops in your code, and inside all of that comparisons happen. Doesn't that suggest a different complexity?